samples/algorithms/chsh-game/Game.qs

```diff
@@ -143,4 +141,22 @@ namespace Microsoft.Quantum.Samples.CHSHGame {
             return MeasureBobQubit(bobBit, bobQubit) == MeasureAliceQubit(aliceBit, aliceQubit);
         }
     }
+
+    /// # Summary
+    /// Plays the optimal classical strategy for the CHSH game: both Alice and Bob both always
+    /// output 0. This should result in success 75% of the time.
+    ///
+    /// # Input
+    /// ## aliceBit
+    /// The bit given to Alice (X).
+    /// ## bobBit
+    /// The bit given to Bob (Y).
+    ///
+    /// # Output
+    /// Whether Alice and Bob's output bits match.
+    function PlayClassicalStrategy(aliceBit : Bool, bobBit : Bool) : Bool {
+        let aliceOutput = false;
+        let bobOutput = false;
+        return aliceOutput == bobOutput;
+    }
```

Comment:
(Just Curiosity)
> function PlayClassicalStrategy(**aliceBit** : Bool, **bobBit** : Bool) : Bool

Doesn't compiler complain that the parameters are not used in the function? If not, then is there a chance that in the the future the compiler or static analyzer will?

In C/C++ if the parameter is not used then it is OK to skip the param's name in the param list (to calm down the compiler/analyzer). Is the same possible in .qs?
> function PlayClassicalStrategy( **: Bool, : Bool**) : Bool

Reply:
> Doesn't compiler complain that the parameters are not used in the function? If not, then is there a chance that in the the future the compiler or static analyzer will?

The Q# compiler doesn't do this currently. I think it would make sense to add that feature though.

> In C/C++ if the parameter is not used then it is OK to skip the param's name in the param list (to calm down the compiler/analyzer). Is the same possible in .qs?

The `: Bool` syntax isn't valid, but what is typically done in ML-style languages like Q# is to prefix or replace the name with an underscore if it's unused. For example, if `aliceBit : Bool` was unused and generated a warning, renaming it to `_aliceBit : Bool` or `_ : Bool` would silence the warning. (Although `_` is not currently allowed in Q# as a parameter name, I think it would make sense to lift that restriction.)